widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx

```diff
@@ -149,6 +171,65 @@ export function WalletList(props: PropTypes) {
                 error={!!error}
               />
             )}
+            {!!experimentalChainWallet && (
+              <Modal
+                open={!!experimentalChainWallet && showExperimentalChainModal}
+                container={modalContainer}
+                onClose={() => {
+                  setExperimentalChainWallet(null);
+                }}>
+                <MessageBox
+                  title={`Add ${blockchainDisplayName} Chain`}
```

Comment:
Please use translated strings for title and description and use 'Blockchain' word.
There are several similar cases in the following code as well.

Reply:
it will be fixed in RF-881